source/upgrade/important-upgrade-notes.rst

```diff
@@ -17,6 +17,19 @@ Important Upgrade Notes
 +----------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | If you’re upgrading from a version earlier than... | Then...                                                                                                                                                          |
 +====================================================+==================================================================================================================================================================+
+| v9.11                                              | Added support for Elasticsearch v8. Also added Beta support for Opensearch. A new config setting ``ElasticsearchSettings.Backend`` has been added to             |
```

Comment:
Please add a line that "For AWS customers on Opensearch, do not enable 'compatibility mode'" and mention it in the changelog as well.